client/state/notices/README.md

```diff
@@ -42,16 +42,28 @@ this.props.successNotice(
 * `errorNotice()`
 * `infoNotice()`
 * `warningNotice()`
+* `updateNotice()`
 
 ## Options
 
 The first argument is the text to be displayed on the notice. The second argument is an optional object and accepts some properties:
 
 *All these parameters are optional*
 
-* `id` (default generated `uniqueId()` ) ID for your notice
+* `id` (default generated `uniqueId()` ) ID for your notice. If a notice with the same ID already exists then the new one will replace it.
```

Comment:
Can we add an example about notices replacing each other? Maybe not with highest priority and I am ok leaving for another PR.